resources/DirectoryLister.php

```diff
@@ -61,7 +61,35 @@ public function __construct() {
         $this->_themeName = $this->_config['theme_name'];
 
     }
-
+	/**
+	* If it is has password enabled
+	* 
+	* @return true if enabled, false if not
+	* @access public
+	*/
+	public function isPasswordProtected () {
+		if ($this->_config['protected_by_password']) {
+		return true;
+		} else {
+		return false;
+		}
+	}
+	/**
+	* Check if the password is correct through sha1 checks
+	* 
+	* @return true if password is valid
+	*		  false if password no match
+	* @access public
+	*/
+	public function checkPassword($password) {
+	$passcheck = sha1($this->_config['password']);
+	$passed = sha1($password);
```

Comment:
Why did you sha1() the passwords here?  I understand that hashing a password is necessary to protect passwords at rest but the password is stored in plain text in the config.  Hashing the passwords here does nothing but add unnecessary overhead to the check password operation.